Sources/Web3Modal/Core/Web3ModalClient.swift

```diff
@@ -276,7 +281,12 @@ public class Web3ModalClient {
             break
         }
     }
-    
+
+    public func removeSessionAndAccount() {
+        store.session = nil
+        store.account = nil
+    }
```

Comment:
Why is this here?

Reply:
@mooshee My mistake, just a test function. This should be removed.